src/main/groovy/life/qbic/datamodel/dtos/business/CostEstimate.groovy

```diff
@@ -0,0 +1,147 @@
+package life.qbic.datamodel.dtos.business
+
+import life.qbic.datamodel.accounting.ProductItem
+import life.qbic.datamodel.dtos.general.Person
+
+/**
+ * A cost estimate for a project
+ *
+ * During project planning a cost estimate is often required to list expected costs.
+ * This estimate is not intended to be legally binding. It provides an overview over the project
+ * scope and the costs associated with it.
+ *
+ * @since: 1.12.0
+ *
+ */
+class CostEstimate {
+
+    /**
+     * Date on which the Cost Estimate was lastly modified
+     */
+    final Date modificationDate
+    /**
+     * The date on which the Cost Estimate expires
+     */
+    final Date expirationDate
+    /**
+     * The customer for which this Cost Estimate was created
+     */
+    final Customer customer
+    /**
+     * The QBiC project manager who was assigned to the project
+     */
+    final Person projectManager
+    /**
+     * The title of the project
+     */
+    final String projectTitle
+    /**
+     * A short description of the project
+     */
+    final String projectDescription
+    /**
+     * A list of items for which the customer will be charged
+     */
+    final List<ProductItem> items
+    /**
+     * The total price of the Cost Estimate (the price of all items)
+     */
+    final double totalPrice
+    /**
+     * The identifier for the Cost Estimate which makes it distinguishable from other Cost Estimate
+     */
+    final CostEstimateId identifier
+    /**
+     * The affiliation of the customer selected for this Cost Estimate
+     */
+    final Affiliation selectedCustomerAffiliation
+
+    static class Builder {
+
+        Date modificationDate
+        Date expirationDate
+        Customer customer
+        ProjectManager projectManager
+        String projectTitle
+        String projectDescription
+        List<ProductItem> items
+        double totalPrice
+        CostEstimateId identifier
+        Affiliation selectedCustomerAffiliation
+
+        Builder(Date modificationDate) {
```

Comment:
I think the modification date is not really relevant if you think about transferring offer information from the view to the use case. I would suggest having the project title, project description, customer, customer affiliation, PM, and items

Reply:
Alright I can do that 👍